BE/app/services/agents/strategy_scraper.py

```
@@ -0,0 +1,165 @@
+        if strategies is None:
+            # TODO: 웹 스크래핑 구현 (Phase 2)
+            self.logger.info("Web scraping not implemented yet")
+            return 0
+        
+        saved_count = 0
+        for strategy_data in strategies:
+            try:
+                await self._save_to_db(session, strategy_data)
+                saved_count += 1
+                self.logger.info(f"Saved strategy: {strategy_data['id']}")
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)